auth0-js/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Adrian Chia <https://github.com/adrianchia>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace auth0 {
+declare module auth0 {
```

Comment:
@aozgaa exporting the namespace seems resolved the issue. Could you please review?